src/view-locator.js

```diff
@@ -50,6 +50,18 @@ export class ViewLocator {
       value = value.constructor;
     }
 
+    // static view strategy
+    if (value.view) {
```

Comment:
This is a breaking change.
Use a symbol instead?

Reply:
You mean a static `view` method / field on a class detection is a breaking change ?

The reason it went with `static resource / view` instead of symbol is that devs can easily create a custom element, without importing anything from Aurelia. I'm not sure about symbol here. Maybe @EisenbergEffect wants to discuss too.

Reply:
I mean that existing code that contains classes with a `view` static field will trigger this code with unintended effects.

Unless we are sure that existing code will not execute this (e.g. if you need to use new API entry points)?

Reply:
I'm using `static view;` in the VMs I feed to my custom `<compose>`like elements - just a feedback for possible scenarios this could be found now.
I do like the `symbol` idea. We could use `Symbol.for()`, define and export the string keys in Aurelia. Now if you want to import the keys you can, or just type them yourself.

Reply:
@StrahilKazlachev That's a nice idea. We can do `Symbol.for('au.view')`, but then I think it kind of defeats the point of having a static method / field. Maybe view is too generic, lets make it `viewStrategy` ? @EisenbergEffect @jods4 ?

Reply:
@bigopon is this really a regression or not? I mean: will existing applications go through this code, or do you have to use a new method to enable this?
From the rest of the changes, I have the impression that this has precedence over conventions, so existing classes _will_ go through this, right?

`viewStrategy` is less likely to be used than `view`... but it's more cumbersome for users and there's always a chance someone uses it already. Something like `au-view` is both shorter and _very_ unlikely to be used (if only because of the dash!).

`Symbol` is made to avoid clashes. But it's even less convenient to use:
```ts
class MyElement {
  [Symbol.for("au-view")]() { return "<div>Hello</div>"; }
}

// Alternative
import { view } from "aurelia-symbols";

class MyElement {
  [view]() { return "<div>Hello</div>"; }
}
```

That said when it comes to convenience I would probably prefer a decorator and then the key/symbol doesn't matter.